src/components/App.js

```diff
@@ -1,10 +1,10 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
```

Comment:
`// @flow`

Reply:
is it necessary to do some kind of flow checking in App.js? it's just a wrapper for the application, so I thought it was unnecessary.

Reply:
@gabrielreisn that's ok :)